editor/sidebar/post-author/index.js

```diff
@@ -12,7 +12,7 @@ import { PostAuthor as PostAuthorForm, PostAuthorCheck } from '../../components'
 export function PostAuthor() {
 	return (
 		<PostAuthorCheck>
-			<PanelRow>
+			<PanelRow className="editor-post-author">
```

Comment:
Since this class is no longer being styled, I'd suggest removing this change ([YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it)).

Reply:
Done.